### DISCUSSION TITLE
question: SD15

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#3024] [Q&A]
if i choose sd15 model and generate

NotImplementedError: You have selected base model other than SDXL. This is not supported yet.

but  in the line "refiner" i see  "Refiner (SDXL or SD 1.5)"
what ist right now  ^^

btw why no sd15 ... ist that such as much different and has more errors ?

**Comment by ILikeToasters** [comment]
You need to use an SDXL model in the base model. You can then use a SD 1.5 model in the refiner. If you want to use SD 1.5 you will need to add a 1.5 model to your checkpoint folder. Fooocus was designed to work with SDXL but you can use it the way I just mentioned if you want.

**Comment by mashb1t** [comment]
see https://github.com/lllyasviel/Fooocus/discussions/3016
